src/pages/auth/TwoFactor.tsx

```diff
@@ -0,0 +1,120 @@
+import backgroundImage from '../../assets/register-login-background.svg';
+import otherLogo from '../../assets/otherLogo.svg';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import useToast from '../../hooks/useToast';
+import useToken from '../../hooks/useToken';
+import { verifyOTP } from '../../redux/features/OTPSlice';
+import { useAppDispatch, useAppSelector } from '../../redux/hooks/hooks';
+import { OTPSchema, OTPSchemaType } from '../../validations/OTP.validation';
+import { DynamicData } from '../../@types/DynamicData';
+import useNProgress from '../../redux/hooks/useNprogress';
+import FormInput from '../../components/Forms/InputText';
+import IconLoader from '../../components/Loaders/IconLoader';
+import { ChevronLeft } from 'lucide-react';
+
+const TwoFactorAuth = () => {
+	const navigate = useNavigate();
+	const handleNavigation = () => {
+		navigate('/');
+	};
+	const [searchParams] = useSearchParams();
+
+	const token = searchParams.get('token');
+	const dispatch = useAppDispatch();
+	const { isLoading } = useAppSelector((state) => state.otp);
+	const { showErrorMessage, showSuccessMessage } = useToast();
+	const { saveAccessToken } = useToken();
+
+	const {
+		register,
+		handleSubmit,
+		formState: { errors },
+	} = useForm<OTPSchemaType>({ resolver: zodResolver(OTPSchema) });
+
+	useNProgress();
+	const onSubmit: SubmitHandler<OTPSchemaType> = async (
```

Comment:
why do we have this useNProgress() hook ? since we have it configured global on app root as wrapper.

Reply:
Forgot to remove it, but now it is removed